migrations/0046_verified_wallet_outbox.up.sql

```diff
@@ -0,0 +1,6 @@
+create table verified_wallet_outbox (
```

Comment:
suggest we include the uuidv5 "linking_id" and the custodian in this table and the deposit_destination, it can act as a log of sorts.

Reply:
I was thinking of this as a very temp table to gives us transactionality a purely technical thing and nothing more, the rows get deleted after they are processed so not sure it would add much. Are you thinking of this as something more? Down the line this will move to events once we can schedule the refactor time

Reply:
i think with 2.5 we don't need a log of linkings, as each profile can only link once.  nvm